Packs/EDL/Playbooks/playbook-Block_Domain_-_External_Dynamic_List_README.md

```diff
@@ -0,0 +1,33 @@
+This playbook blocks domains using External Dynamic Link.
+The playbook adds a tag to the inputs domain indicators. the tagged indicators can be publish as External Dynamic list that can be added to blocklist using products like Panorama by Palo Alto Networks.
```

Comment:
You should update the readme file :) 
as your description updated, should update here as well 
(also the release notes)

Reply:
Done